dockerfiles/piraeus-server/entry.sh

```diff
@@ -1,5 +1,31 @@
 #!/bin/bash
 
+set -e
+
+run_migration() {
+	if /usr/share/linstor-server/bin/linstor-config all-migrations-applied --logs=/var/log/linstor-controller --config-directory=/etc/linstor "$@" ; then
+		return 0
+	fi
+
+	VERSION=$(/usr/share/linstor-server/bin/linstor-config run-migration -v)
+
+	BACKUP_NAME=${BACKUP_NAME:-linstor-backup-for-$(hostname)}
+
+	if ! kubectl get secrets "${BACKUP_NAME}"; then
```

Comment:
Are we relying on this running in a pod with a randomly generated `hostname`?

Reply:
That was my intention. The issue we actually want to solve is: How to create a backup for every upgrade without creating backups on every failed upgrade attempt.
* Using random names would lead to duplicate backups every time the migration is restarted
* Using a name based on the target version would mean we potentially only back up once. Which would be bad if the user manually downgraded and tried to run the upgrade again with slightly modified data.

I settled on the pod name to have something "random" but stable for one migration.